test/functional/DDR_Test/playlist.xml

```diff
@@ -130,4 +130,154 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		</impls>
 	</test>
 
+	<test>
+		<testCaseName>testDDRExtJunit_MonitorsAndDeadlock_1</testCaseName>
```

Comment:
Could you remove `_` before the number? i.e., rename `testDDRExtJunit_MonitorsAndDeadlock_1 ` to `testDDRExtJunit_MonitorsAndDeadlock1`. `testDDRExtJunit_MonitorsAndDeadlock_1 ` will cause confusion as TKG adds suffix `_<number>` to test name as well. Please also change the rest of tests in this PR.